Source/MBF.Displays.OLED1Xplained.pas

```diff
@@ -0,0 +1,704 @@
+unit MBF.Displays.OLED1Xplained;
```

Comment:
This file will need reworking, it uses up too much memory by defining fonts and some screenbuffer content.
Please check my ssd1963 code, fonts are provided via extra units. 
I have also written a (very crude) windows GUI that allows you to convert ttf fonts to pascal fonts.
Please check this site: 
https://int10h.org/oldschool-pc-fonts/fontlist/
They have some nice ttf fonts that can be converted to perfectly looking pascal fonts with my tool. 
I only need to extend the tool to now also create fonts in format specific to those OLED Displays